docs/userdocs/k8s/README.md

```diff
@@ -22,19 +22,40 @@ Submarine for K8s supports distributed TensorFlow and PyTorch.
 
 Submarine can run on K8s >= 1.14, supports features like GPU isolation.
 
+We have validated Submarine on the following versions:
+
+[FIXME]: is it accurate?
```

Comment:
@tangzhankun , @jiwq , @pingsutw : What we should put for supported versions?

Reply:
LGTM, I had verified 14/15/16 version.